travvy-server/src/config/config.js

```diff
@@ -1,6 +1,6 @@
 
 module.exports = {
-  port: 3306,
```

Comment:
realised this has to be the port on which you're hosting your local

Reply:
ahhh ok makes sense, so not the port for the actual database?

Reply:
im glad you were able to figure this one out, it was causing us so many problems connecting to the DB

Reply:
makes sense!!